.github/workflows/build.yml

```diff
@@ -126,15 +126,16 @@ jobs:
         uses: MansaGroup/nrwl-nx-action@v2
         with:
           targets: "build"
-          projects: "winglang,sdk,wing-api-checker,jsii-docgen,wing-playground"
+          all: true
           args: "--configuration=release --output-style=stream --verbose"
 
       # wingc require wingsdk to be built first in order to run
       - name: Test
         uses: MansaGroup/nrwl-nx-action@v2
         with:
           targets: "test"
-          projects: "wingc,sdk"
+          all: true
```

Comment:
this will cause wingc to do a full native build. I would suggest adding the specific new projects you want here instead.

Reply:
the test target for wingc runs cargo test, which uses the native platform target by default

Reply:
As discussed, we are already running `wingc` tests.